site/topics/logic/boolean-logic.rst

```diff
@@ -95,6 +95,114 @@ Truth Tables
 Properties of Logical Operators
 ===============================
 
+* There are several algebraic properties that hold for boolean logic
+* Most of these are intuitive, but will be presented here for completness
+
+
+.. list-table:: Boolean Algebra Laws
+    :widths: auto
+    :align: center
+
+    * - Identity for :math:`\lor`
+      - :math:`a \lor false = a`
+    * - Identity for :math:`\land`
+      - :math:`a \land true = a`
+    * - Idempotence for :math:`\lor`
+      - :math:`a \lor a = a`
+    * - Idempotence for :math:`\land`
+      - :math:`a \land a = a`
+    * - Annihilator for :math:`\lor`
+      - :math:`a \lor true = true`
+    * - Annihilator for :math:`\land`
+      - :math:`a \land false = false`
```

Comment:
I've never heard these called annihilators. In my head that's still a differential equation solution approach. I'd maybe write these as _identity for true/false_ instead, unless annihilator is a newfangled term the kids are using these days.

Reply:
I'm glad you said this as that's what my brain said too. However. wikipedia goes with annihilator, so I opted to go with that wording \shrug 

![image](https://github.com/jameshughes89/cs2XX-ComputerArchitecture/assets/1328048/e31f3850-9275-41a2-bc60-710f4f67e6d9)